h/views/api/config.py

```diff
@@ -4,6 +4,17 @@
 from h.util import cors
 
 
+#: Decorator that adds CORS headers to API responses.
```

Comment:
I saw the Slack thread this morning about attr comments and presume you're doing this right (I frankly have no idea and would be copying/pasting from other examples in code myself).

Reply:
Yes. The TL;DR is that this is consistent with docstrings for attributes in other h code but Python does support standard `"""` docstrings for attributes which are used in other projects.